content/posts/newsletter/2021-02-05.md

```diff
@@ -0,0 +1,184 @@
+
+
+## Extensions and specifications
+
+Developers and other standards experts from around the world collaborate on these extensions, developing new specifications for emerging practices, and refining existing ways of doing things. Proposed by anybody, the particularly successful ones end up as Final or Active - depending on their type - while others are carefully archived as Deferred. This life cycle is described in [XEP-0001](https://xmpp.org/extensions/xep-0001.html), which contains the formal and canonical definitions for the types, states, and processes. [Read more about the standards process](https://xmpp.org/about/standards-process.html).
+
+### Proposed Extensions
+
+The XEP development process starts by writing up an idea and submitting it to the XMPP Editor. Within two weeks, the Council decides whether to accept this proposal as an Experimental XEP.
+
+-   Several extensions have been proposed this month. They have been accepted by the council, and are listed in the New section below.
+
+### New
+
+-   Version 0.1.0 of [XEP-0449](https://xmpp.org/extensions/xep-0449.html) (Stickers)
+    -   This specification provides a protocol to send stickers and to create and share sticker packs.
+    -   Accepted by vote of Council on 2020-11-18.
+
+-   Version 0.1.0 of [XEP-0448](https://xmpp.org/extensions/xep-0448.html) (Encryption for stateless file sharing)
+    -   This specification provides a protocol for sharing encrypted files using the stateless file sharing protocol (XEP-0447).
+    -   Accepted by vote of Council on 2020-11-18.
+
+-   Version 0.1.0 of [XEP-0447](https://xmpp.org/extensions/xep-0447.html) (Stateless file sharing)
+    -   This specification describes a protocol for stateless asynchronous file sharing with integrity and transport flexibility. It allows clients to provide a good interoperable user experience in combination with Carbons and MAM.
+    -   Accepted by vote of Council on 2020-11-18.
+
+-   Version 0.1.0 of [XEP-0446](https://xmpp.org/extensions/xep-0446.html) (File metadata element)
+    -   This specification defines a generic file metadata element to be used in other specifications.
+    -   Accepted by vote of Council on 2020-11-18.
+
+-   Version 0.2.0 of [XEP-0445](https://xmpp.org/extensions/xep-0445.html) (Pre-Authenticated In-Band Registration)
+    -   This document extends the In-Band-Registration protocol to use invitation tokens, e.g. for registering accounts on non-public servers.
+    -   Accepted by vote of Council on 2020-11-04.
+
+### Deferred
+
+If an experimental XEP isn’t updated for more than six months, it will be moved off Experimental to Deferred. If there is another update, it will put the XEP back onto Experimental.
```

Comment:
```suggestion
If an experimental XEP isn’t updated for more than twelve months, it will be moved off Experimental to Deferred. If there is another update, it will put the XEP back onto Experimental.
```
See https://xmpp.org/extensions/xep-0001.html#states-Deferred

Reply:
@wurstsalat3000 approved?

Reply:
I’m not sure, https://xmpp.org/about/standards-process.html says 6 months.

Reply:
@melvo

Reply:
@wurstsalat3000 I created #852 for that.

Reply:
@melvo I see, thanks for the hint! :+1:

Reply:
I will close the discussion and reference to the PR #852 for further points.